Sources/Utils/HandlerRegistryService.swift

```diff
@@ -52,16 +52,29 @@ class HandlerRegistryService {
         
         let binderToUse = binders.property?[skLocal] ?? binders.property?[skGlobal]
         
+        func updateBinder(b : BinderProtocol) {
+            if binders.property?[skLocal] != nil {
+                binders.property?[skLocal] = b
+            } else {
+                binders.property?[skGlobal] = b
+            }
```

Comment:
Don't we need synchronization to update binders dictionary?
Let's fix this unsafe dictionaries/arrays in other places as well.

Reply:
the binder set is an atomic property.  the binder is a copy and so this is an atomic operation.  the get and set are separate.  but, the again, since this is only an issue with a singleton that usually provides a instance with creation. it should not be an issue.  i don't want to slow everything down by constantly wrapping everything in a dispatch.